test/parallel/test-module-readonly.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const cp = require('child_process');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (common.isWindows) {
+  // Create readOnlyMod.js and set to read only
+  const readOnlyMod = path.join(tmpdir.path, 'readOnlyMod');
+  const readOnlyModRelative = path.relative(__dirname, readOnlyMod);
+  const readOnlyModFullPath = readOnlyMod + '.js';
```

Comment:
We usually prefer template literals in such cases, so maybe:
```
const readOnlyModFullPath = `${readOnlyMod}.js`;
```
but feel free to ignore this and next stylistic notes)